### PR TITLE
Visitor can read specific article

### PR DESCRIPTION
**Pull request by Joberg95**
#### Description
Added article links which leads to that specific article page.
Added a "back" button on the specific article page which returns you to index.

#### What I have learned working on this feature: 
How to properly create articles in the rails console (I forgot how to do that)
## Index:
image

## Article page with back button:
image

### CODE REVIEW

File: features/step_definitions/assertions_steps.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Then("I should see {string}") do |content|
     expect(page).to have_content content
+end
+
+Then("I should see the {string} page") do |page|
```
Comment (magnus-thor) [inline features/step_definitions/assertions_steps.rb]: you should not use page as a variable name, page object points to the page you are on and with this test you asking if `page = page`

File: app/views/articles/index.html.erb [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
-    </p>
-    <p>
-        <%= article.content %>
-    </p>
-<% end %> 
\ No newline at end of file
+<h1> The Viking Times </h1>
+
+<% @articles.each do |article|%>
+    <%= link_to article.title, article_path(article) %>
```
Comment (magnus-thor) [inline app/views/articles/index.html.erb]: Are you only showing the article title on index page?

Comment (Joberg95) [reply app/views/articles/index.html.erb]: fixed

File: features/step_definitions/assertions_steps.rb (hunk 1 above)
Comment (Joberg95) [reply features/step_definitions/assertions_steps.rb]: done

**Review by NiciP — APPROVED** [review]

**Review by aleciabenjamin — APPROVED** [review]

**Review by perhegg — APPROVED** [review]

**Review by magnus-thor — APPROVED** [review]
good job.

**Pull request merged**